private-templates-service/client/src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
You're welcome to define your own patterns for this - but the way I set up my components, the file structure looks like this:

/components/Dashboard
            - index.ts
            - Dashboard.ts
            - DashboardTests.ts

Then the Dashboard.ts would be this, and the index.ts file would be a super simple file like:
`import Welcome from "./Welcome";

export default Welcome `

Then when you use Welcome, you can import from "./components/Welcome" instead of "./components/Welcome/Welcome"

Reply:
Not sure if I understand this pattern. Could you elaborate?

Reply:
Here is an example of a component library using this structure: https://github.com/pixeloven/typescript-react-component-library/tree/master/src/shared/components/atoms/Button